tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -1312,10 +1314,11 @@ lldbg_create_ftn_subrange_via_sdsc(LL_DebugInfo *db, int findex, SPTR sptr,
 }
 
 static void
-lldbg_get_bounds_for_sdsc(LL_DebugInfo *db, int findex, SPTR sptr, int rank,
-                          LL_MDRef *lbnd_expr_mdnode,
-                          LL_MDRef *ubnd_expr_mdnode,
-                          LL_MDRef *stride_expr_mdnode)
+lldbg_get_bounds_for_sdsc(LL_DebugInfo *db, int findex, SPTR sptr,
+                                      int rank, LL_MDRef *count_expr_mdnode,
+                                      LL_MDRef *lbnd_expr_mdnode,
+                                      LL_MDRef *ubnd_expr_mdnode,
+                                      LL_MDRef *stride_expr_mdnode)
```

Comment:
Could you left-align the arguments?

Reply:
Thanks for your comment. Done in latest revision.